File: src/main/java/no/nav/aap/api/søknad/pdl/PDLHealthIndicator.java
package no.nav.aap.api.søknad.pdl;

import no.nav.aap.api.søknad.health.AbstractPingableHealthIndicator;
import org.springframework.stereotype.Component;

    @Component
    public class PDLHealthIndicator extends AbstractPingableHealthIndicator {
        public PDLHealthIndicator(PDLWebClientAdapter adapter) {
            super(adapter);
        }
}

File: src/main/kotlin/no/nav/aap/api/søknad/pdl/PDLHealthIndicator.kt
package no.nav.aap.api.søknad.pdl

import no.nav.aap.api.søknad.health.AbstractPingableHealthIndicator
import org.springframework.stereotype.Component

@Component
class PDLHealthIndicator(adapter: PDLWebClientAdapter) : AbstractPingableHealthIndicator(adapter!!)